packages/web/src/store/sagas.ts

```diff
@@ -136,7 +135,7 @@ export default function* rootSaga() {
     walletSagas(),
 
     // Cast
-    castSagas(webStoreContext),
+    castSagas(storeContext),
```

Comment:
We could probably also refactor cast sagas to use the context

Reply:
def!